doc/devel/MEP/MEP28.rst

```diff
@@ -165,6 +160,19 @@ users to migrate to a new API.
 Deprecation of ``sym`` however, may be have a much broader reach into
 the matplotlib userbase.
 
+An accelerated timeline could look like the following:
+
+  1. v2.0.1 add transforms to ``cbook.boxplots_stats``, expose in ``Axes.boxplot``
+  2. v2.1.0 deprecate ``usermedians``, ``conf_intervals``, ``sym`` parameters
+  3. v2.2.0 make deprecations noisier (???)
+  3. v2.3.0 remove ``usermedians``, ``conf_intervals``, ``sym`` parameters
+  4. v2.3.0 deprecate ``notch`` in favor of ``shownotches`` to be consistent with
```

Comment:
Which did you have in mind? that was my intention with moving from `notch` to `shownotches`